### ISSUE TITLE
Login redirect doesn't work when web app is installed in ROOT context

### ISSUE THREAD

**Issue by yrro** [irods-contrib/metalnx-web#202]
I tried installing Metalnx into the `ROOT` context so that it lives at `/` rather than at `/metalnx/`; but I found that the login redirect still starts with `/metalnx/`:

```
$ oc exec metalnx-7479b67dd6-kzbf6  -- curl -sS -I http://localhost:8080/
HTTP/1.1 302
Location: /metalnx/login/exception/
```

This should be `/login/exception/`.

**Comment by trel** [comment]
Understood.

I'm pretty sure the string 'metalnx' is hard-coded into a route or path in hundreds of places in this codebase.

It would be much simpler/faster to route from `/` to `/metalnx/` if you need to have the application appear at the root.

**Comment by michael-conway** [comment]
Yes, for now I think we have to live with the /metalnx mapping, over the medium term much of the login and thymeleaf based handling will go away as it moves more towards a single page proper javascript->AJAX arrangment. My two cents.
